.github/workflows/kiro-integration.yml

```
@@ -165,16 +232,16 @@
       contents: write
       issues: write
       pull-requests: write
-      
+
     steps:
     - name: Checkout repository
       uses: actions/checkout@v4
-      
+
     - name: Setup Python
       uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)